// legion-ui/src/components/Alert.tsx
/** @jsx jsx */

import React from 'react';
import { jsx, useThemeUI } from '@theme-ui/core';
import { get } from '@theme-ui/css';
import {
  Alert as Component,
  AlertProps as Props,
  Box,
  Close,
  Flex,
  ForwardRef,
  Text,
} from '@theme-ui/components';

export interface AlertProps extends Props {
  icon?: React.ReactNode;
  onAlertClose?: React.MouseEventHandler<HTMLButtonElement>;
  variant?: 'primary' | 'secondary' | 'info' | 'warning' | 'success' | 'error';
}

export const Alert: ForwardRef<HTMLDivElement, AlertProps> = React.forwardRef(
  ({ children, icon, onAlertClose, variant = 'primary', ...rest }, ref) => {
    const context = useThemeUI();
    const variants = get(context.theme, `alerts.${variant}`);
    return (
      <Component ref={ref} sx={{ ...variants }} {...rest}>
        <Flex
          sx={{
            alignItems: icon ? 'flex-end' : 'center',
            flexDirection: 'row',
            justifyContent: 'space-between',
            width: '100%',
          }}
        >
          {icon ? (
            <Box
              sx={{
                display: 'flex',
                alignItems: 'center',
                justifyContent: 'center',
              }}
            >
              <span>{icon}</span>
              <Text
                sx={{
                  marginLeft: '10px',
                }}
              >
                {children}
              </Text>
            </Box>
          ) : (
            <Text>{children}</Text>
          )}
          <Close
            sx={{
              cursor: 'pointer',
            }}
            onClick={onAlertClose}
          />
        </Flex>
      </Component>
    );
  }
);
