// legion-ui/src/components/Radio.tsx
/** @jsx jsx */

import React from 'react';
import {
  jsx,
  // useThemeUI
} from '@theme-ui/core';
// import { get } from '@theme-ui/css'
import { Radio as Component, RadioProps as Props, ForwardRef } from '@theme-ui/components';

export interface RadioProps extends Props {
  // checked?: boolean,
  // disabled?: boolean,
  // label?: '' | String,
  // sizeBox?: 'lg' | 'sm' | String,
  // variant?: 'check' | 'indeterminate',
}

export const Radio: ForwardRef<HTMLInputElement, RadioProps> = React.forwardRef(
  ({ ...rest }, ref) => {
    // const context = useThemeUI();
    // const checkmark = get(context.theme, `forms.radio.checkmark`);

    return (
      <div sx={{
        position: 'relative'
      }}>
        <Component
          {...rest}
          ref={ref}
          sx={{
            'input:checked ~ &': {
              // border: '2px solid #E42313',
              border: '.5px solid #E42313',
              // background: '#FFFFFF',
              color: '#FFFFFF',
              background: '#E42313',
              // '::after ~ .css-st8bg9': {
              // width: '20px',
              // height: '20px',
              // background: 'yellow',
              // zIndex: 1,
              // display: 'block'
              // }
            },
          }}
        />
        {/* <div
          // sx={{ ...checkmark }}
          sx={{
            width: '20px',
            height: '20px',
            background: 'yellow',
            position: 'absolute',
            display: 'none'
          }}
        /> */}
      </div>
    );
  }
)

export default Radio;
