// legion-ui/src/foundation/typography.ts
export const text = {
  body_lg_reguler: { fontSize: '18px', lineHeight: '28px', fontWeight: 400 },
  body_lg_semibold: { fontSize: '18px', lineHeight: '28px', fontWeight: 600 },
  body_lg_bold: { fontSize: '18px', lineHeight: '28px', fontWeight: 700 },
  body_lg_italic: {
    fontSize: '18px',
    lineHeight: '28px',
    fontWeight: 400,
    fontStyle: 'italic',
  },
  body_lg_underline: {
    fontSize: '18px',
    lineHeight: '28px',
    fontWeight: 400,
    textDecoration: 'underline',
  },
  body_lg_underline_semibold: {
    fontSize: '18px',
    lineHeight: '28px',
    fontWeight: 600,
    textDecoration: 'underline',
  },
  body_lg_underline_bold: {
    fontSize: '18px',
    lineHeight: '28px',
    fontWeight: 700,
    textDecoration: 'underline',
  },
  body_sm_reguler: { fontSize: '16px', lineHeight: '24px', fontWeight: 400 },
  body_sm_semibold: { fontSize: '16px', lineHeight: '24px', fontWeight: 600 },
  body_sm_bold: { fontSize: '16px', lineHeight: '24px', fontWeight: 700 },
  body_sm_italic: {
    fontSize: '16px',
    lineHeight: '24px',
    fontWeight: 400,
    fontStyle: 'italic',
  },
  body_sm_underline: {
    fontSize: '16px',
    lineHeight: '24px',
    fontWeight: 400,
    textDecoration: 'underline',
  },
  body_sm_underline_semibold: {
    fontSize: '16px',
    lineHeight: '24px',
    fontWeight: 600,
    textDecoration: 'underline',
  },
  body_sm_underline_bold: {
    fontSize: '16px',
    lineHeight: '24px',
    fontWeight: 700,
    textDecoration: 'underline',
  },

  caption_lg_reguler: { fontSize: '14px', lineHeight: '21px', fontWeight: 400 },
  caption_lg_semibold: {
    fontSize: '14px',
    lineHeight: '21px',
    fontWeight: 600,
  },
  caption_lg_bold: { fontSize: '14px', lineHeight: '21px', fontWeight: 700 },
  caption_lg_italic: {
    fontSize: '14px',
    lineHeight: '21px',
    fontWeight: 400,
    fontStyle: 'italic',
  },
  caption_lg_underline: {
    fontSize: '14px',
    lineHeight: '21px',
    fontWeight: 400,
    textDecoration: 'underline',
  },
  caption_lg_underline_semibold: {
    fontSize: '14px',
    lineHeight: '21px',
    fontWeight: 600,
    textDecoration: 'underline',
  },
  caption_lg_underline_bold: {
    fontSize: '14px',
    lineHeight: '21px',
    fontWeight: 700,
    textDecoration: 'underline',
  },
  caption_sm_reguler: { fontSize: '12px', lineHeight: '18px', fontWeight: 400 },
  caption_sm_semibold: {
    fontSize: '12px',
    lineHeight: '18px',
    fontWeight: 600,
  },
  caption_sm_bold: { fontSize: '12px', lineHeight: '18px', fontWeight: 700 },
  caption_sm_italic: {
    fontSize: '12px',
    lineHeight: '18px',
    fontWeight: 400,
    fontStyle: 'italic',
  },
  caption_sm_underline: {
    fontSize: '12px',
    lineHeight: '18px',
    fontWeight: 400,
    textDecoration: 'underline',
  },
  caption_sm_underline_semibold: {
    fontSize: '12px',
    lineHeight: '18px',
    fontWeight: 600,
    textDecoration: 'underline',
  },
  caption_sm_underline_bold: {
    fontSize: '12px',
    lineHeight: '18px',
    fontWeight: 700,
    textDecoration: 'underline',
  },

  h1: { fontSize: '60px', lineHeight: '72px', fontWeight: '700' },
  h2: { fontSize: '48px', lineHeight: '56px', fontWeight: '700' },
  h3: { fontSize: '36px', lineHeight: '44px', fontWeight: '700' },
  h4: { fontSize: '30px', lineHeight: '40px', fontWeight: '700' },
  h5: { fontSize: '24px', lineHeight: '36px', fontWeight: '700' },
  h6: { fontSize: '20px', lineHeight: '28px', fontWeight: '700' },
};
